ArduPlane/navigation.cpp

```diff
@@ -149,11 +149,10 @@ void Plane::calc_airspeed_errors()
     }
 
     // low pass filter speed scaler, with 1Hz cutoff, at 10Hz
-    const float speed_scaler = calc_speed_scaler();
+    const Vector3f speed_scaler = calc_rpy_speed_scaler();
     const float cutoff_Hz = 2.0;
     const float dt = 0.1;
-    surface_speed_scaler += calc_lowpass_alpha_dt(dt, cutoff_Hz) * (speed_scaler - surface_speed_scaler);
-
+    surface_speed_scaler_rpy += (speed_scaler - surface_speed_scaler_rpy) * calc_lowpass_alpha_dt(dt, cutoff_Hz);
```

Comment:
this filter may be enough for the accel noise?

Reply:
As per previous comment I will filter airspeed and longitudinal accel separately before combining in the prop wash velocity calculation.